packages/rich-text-resolver-markdown/src/markdown.ts

```diff
@@ -49,6 +49,19 @@ const listResolver = (level: number, index: number) =>
     (str) => (level === 1 ? `${str}\n\n` : str),
   ]);
 
+export const sanitizeWhiteSpaces = (str: string, mark = "**"): string => {
```

Comment:
You always supply the second parameter. Remove the default to ensure all usages are explicit.